### ISSUE TITLE
Coverage reports are empty.

### ISSUE THREAD

**Issue by olegskl** [kt3k/karma-browserify-isparta-example#1]
I might be doing something wrong, but the reports seem to be empty.

**Comment by kt3k** [comment]
Sounds very strange. Did you just clone this repo and get the empty report? or did anything else?

**Comment by olegskl** [comment]
Yes, I cloned the repo, installed the dependencies and tried running `npm test` and `npm run ci`. Both resulted in empty reports.

**Comment by kt3k** [comment]
Just in case which node and npm versions do you use?
I'll try add some more versions to the CI settings later.

**Comment by olegskl** [comment]
```bash
karma-browserify-isparta-example ➤ node --version
v5.4.0
karma-browserify-isparta-example ➤ npm --version
3.3.12
```
